less/index.less

```
@@ -57,3 +57,178 @@ html, body {
     height: 100%;
 }
 
+html {
+	font-size: 62.5%;
+}
+
+//Variables
+@background-color: lightgray;
+@font-family: Tahoma, Helvetica, serif;
+@height: 100%;
+@padding: 10px;
+@font-size: 4rem;
+@width: 45%;
+@flex-direction: column;
+@justify-content: center;
+@table-background-color: rgb(199, 196, 196);
+
+//Mixins
+.middle-left-right {
+	display: flex;
+	flex-direction: column;
+	width: 50%;
+}
+
```

Great use of a mixin you can create a second one with maybe using flex-direction: row;
```suggestion
//Mixins
.middle-left-right {
	display: flex;
	flex-direction: column;
	width: 50%;
}
```